### PR TITLE
Add Installing via SPM

### PR DESCRIPTION
**Pull request by razor313**
* According to [this issue](https://github.com/SwiftGen/SwiftGen/issues/824), I've added instructions in order to install SwiftGen vie SPM.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -163,6 +163,53 @@ Or add the path to the `bin` folder to your `$PATH` and invoke `swiftgen` direct
...
+
+1. Click on your project in the file list, choose your target under `TARGETS`, click the `Build Phases` tab
+2. Add a `New Run Script Phase` by clicking the little plus icon in the top left
+3. Drag the new `Run Script` phase **above** the `Compile Sources` phase, expand it and paste the following script:
+
+    ```bash
+    cd BuildTools
+    SDKROOT=iphoneos4.0
+    #swift package update #Uncomment this line temporarily to update the version used to the latest matching your BuildTools/Package.swift file
+    swift run -c release --package-path Tools/SwiftFormatter swiftformat --swiftversion "5.3" "$SRCROOT"
```
Comment (lordzsolt) [inline README.md]: Ideally this should show how to use `SwiftGen` and not `SwiftFormatter`

**Review by lordzsolt — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (razor313) [reply README.md]: @lordzsolt Thank you, I've updated the command.

**Comment by ignotusverum** [comment]
@razor313 Doesn't seem like this approach working, at least for me - curious how where we should get `Tools/SwiftFormatter ` from?

**Comment by razor313** [comment]
@ignotusverum I've tested again and it works properly. About the `Tools/SwiftFormatter` it's just a path and you can choose anything that you want for instance you can make a directory which is named SwiftFormatter.

**Comment by razor313** [comment]
It has been supported through #411, therefore I close it. :)

**Comment by djbe** [comment]
To clarify for anyone else, this is NOT "build tools" support that is in [swift evolution](https://github.com/apple/swift-evolution/blob/main/proposals/0303-swiftpm-extensible-build-tools.md). Once that is added, we'll add support for it as well 😄

**Pull request closed without merging**